integration-tests/tests/index.test.ts

```diff
@@ -36,112 +36,198 @@ describe('encrypt and decrypt', async () => {
       table: 'users',
     })
 
-    const decrypted = await decrypt(client, ciphertext.c)
+    const decrypted = await decrypt(client, { ciphertext: ciphertext.c })
 
     expect(decrypted).toBe(originalPlaintext)
   })
 
-  test('can pass in undefined for ctsToken', async () => {
-    const client = await newClient(encryptConfig)
+  test('can explicitly pass in undefined for optional fields', async () => {
+    const client = await newClient({ encryptConfig })
     const originalPlaintext = 'abc'
 
-    const ciphertext = await encrypt(
-      client,
-      {
+    const ciphertext = await encrypt(client, {
+      plaintext: originalPlaintext,
+      column: 'email',
+      table: 'users',
+      serviceToken: undefined,
+      lockContext: undefined,
+    })
+
+    const decrypted = await decrypt(client, {
+      ciphertext: ciphertext.c,
+      lockContext: undefined,
+      serviceToken: undefined,
+    })
+
+    expect(decrypted).toBe(originalPlaintext)
+  })
+
+  test('encrypt throws an error when identityClaim is used without a service token', async () => {
```

Comment:
Adding in a little more coverage (unhappy path for identity claims for various functions).